src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokens5.java

```diff
@@ -1,8 +1,8 @@
-/*
+/* // violation '.*'a href'.*'
```

Comment:
What for we need this text ?

Reply:
We don't need this text. I added it just for consistency across comments in a file.

Reply:
why we need this violation line on the first line?

Reply:
Please remove it, I do not understand this consistency. Header file comment is config for Check, and nothing else.

Reply:
The first line is a violation because the comment token contains the text ```a href```. How can we remove it?

Reply:
Now I get it. Config text contains illegal text so vi violate inlined config comment.